src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc

```diff
@@ -172,6 +179,17 @@ To change the access mode for workspace PVCs, pass the `CHE_INFRA_KUBERNETES_PVC
 
 Another restriction is that only pods in the same namespace can use the same PVC.  The `CHE_INFRA_KUBERNETES_PROJECT` environment variable should not be empty.  It should be either the Che server namespace where objects can be created with the Che service account (SA) or a dedicated namespace where a token or a username and password need to be used.
 
+[id="per-workspace-pvc-strategy"]
+=== PerWorkspace PVC Strategy
```

Comment:
```suggestion
=== Per porkspace PVC strategy
```

Reply:
Will change to `Per Workspace PVC strategy`